taskcluster/ci/config.yml

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
Note for future readers: I see no treeherder config has ever been done on this project. Since it's going into maintenance mode, I don't think it's worth reporting there.